tests/001-test-use-devs-to-create-thin-pool.sh

```diff
@@ -17,6 +17,7 @@ test_devs() {
   cat << EOF > /etc/sysconfig/docker-storage-setup
 DEVS="$devs"
 VG=$vg_name
+CONTAINER_THINPOOL=docker-pool
```

Comment:
All the tests below 100 are docker legacy tests and should work as it is. these are good tests to make sure docker continues to work even if CONTAINER_THINPOOL is not specified.

So I don't think we should be modifying tests below 100. Only new tests should specify CONTAINER_THINPOOL.